src/Microsoft.Health.Fhir.Core/Features/Operations/JobRecordProperties.cs

```diff
@@ -99,6 +99,12 @@ public static class JobRecordProperties
 
         public const string StorageAccountContainerName = "storageAccountContainerName";
 
+        public const string Filters = "filters";
+
+        public const string Filter = "filter";
```

Comment:
One for JobRecord, and another is for JobProgress, but what is the difference?
Also in JobProgress:
why one property is plural and other is singular?
```
      [JsonProperty(JobRecordProperties.Filter)]
        public ExportJobFilter Filter { get; private set; }

        [JsonProperty(JobRecordProperties.FinishedFilters)]
        public bool FinishedFilters { get; private set; }
```
Can this properties be distinguished by more than just a plurality (not sure it's a right word, basically on attribute of how many)

Reply:
Filters is for the entire list of filters for the job. Filter is for the filter that is currently being evaluated, so we can restart in the case of an interrupted job. FinishedFilters is a flag that indicates that all filters have been evaluated and we are now doing the search for any resources that didn't have filters.

Reply:
I'll add comments and see what I can do to change the names.